Sources/WishKit/SwiftUI/WKButton.swift

```diff
@@ -68,18 +68,19 @@ struct WKButton: View {
     }
 
     var textColor: Color {
+        let button = style == .primary ? WishKit.config.buttons.saveButton : WishKit.config.buttons.cancelButton
```

Comment:
If the style is primary it's a save button otherwise a cancel button? That needs at least a comment